webui/react/public/wait/index.html

```diff
@@ -1,29 +1,32 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <meta name="theme-color" content="#000000" />
-    <meta name="description" content="Determined Deep Learning Training Platform" />
-    <link rel="shortcut icon" type="image/x-icon" href="/favicons/favicon.ico" />
-    <link rel="apple-touch-icon" href="/favicons/apple-touch-icon.png" />
-    <link rel="manifest" href="/manifest.json" />
-    <title>Determined AI</title>
-    <link href="/wait/wait.css" rel="stylesheet" />
-  </head>
 
-  <body>
-    <div class="container">
-      <img class="logo" src="/favicons/favicon-192x192.png" />
-      <div class="status">
-        <label>Service State:</label> 
-        <div id="state">Loading...</div>
-      </div>
-      <div id="tip"></div>
-      <div class="spinner">
-        <div id="spinner" class="icon-spinner spin"></div>
-      </div>
+<head>
+  <base href="/det">
```

Comment:
indentaion changed. the real change is /det is prefixed

Reply:
**question:** A bit confused about this change. Didn't you move away from hard coding `/det` to use `PUBLIC_URL`? And either applying a post process of search and replace with the env `PUBLC_URL` or having webpack to do that job during build?

Reply:
Yes I did but then I took it out (and into https://github.com/determined-ai/determined/pull/1559) I took out all of that because it was more code and it would be obsolete when we bring wait page into react. this change alone fixes the reported bug in the ticket.